docs(readme): unify palette & usage (#2005)
catppuccin/catppuccin: README.md

### 🎨 Palette

> **Catppuccin** consists of 4 beautiful pastel color palettes, named **flavours**. All the details can be found below.<br>

Reviewer: ```suggestion
**Catppuccin** consists of 4 beautiful pastel color palettes, named **flavors**. All the details can be found below.<br>
```
We use `color` in the previous sentence so we might as well be consistent here. Although, I'm actually not fond of my changes a few months ago where I tried to use the American spellings for our docs. Catppuccin "leadership" is very European so maybe it's worth just running a command to change it all :P

Author: Oh, see … I'm incerdibly blind! I think we've onced agreed on british spelling as our guideline. I've just noticed that it was american on flavor because it's highlighted in bold.

> If you want to use them for your own project please refer to our [style guide](https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md) for general use cases and guidelines, and [catppuccin/palette](https://github.com/catppuccin/palette) where you can find integrations with popular frameworks and tools.

Reviewer: ```suggestion
If you want to use them for your own project, refer to our [style guide](https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md) for general use cases and guidelines. Additionally, you can find integrations with popular frameworks and tools in [catppuccin/palette](https://github.com/catppuccin/palette).
```

I feel like this reads better. What do you think?

Reviewer 2: I think the wording changes makes it seem more formal. But it comes at the cost of seeming less friendly. Up to you all after all.

Author: I actually don't think it has to sound super friendly and lovely here as we mainly want to make sure that colours are used and applied correctly. Making it more formal accentuates that intend better.

Reviewer: Yeah, I agree, there's something about the 🥺 tone with the `please` that doesn't fit here. If they want to use the palette, they should refer to the style guide. Instead, with the `please`, it just sounds like we're asking them nicely. 

Also instead of using arbitrary friendliness as a metric for this, I'd be much more interested in making sure the readability is good enough to be parsed by as many people as possible, using maybe something like [readable](https://app.readable.com/text/?demo).
